### ISSUE TITLE
[Bug][a11y] Languages are double spoken by the Talkback

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#8101]
#### Prerequisites
- Make sure that you have activated Talkback

## Steps to reproduce
1. Open Firefox Preview;
2. Go to Settings;
3. Open Language section;
4. Select different languages to be read by Talkback;

### Expected behavior
All languages are read correctly by the Talkback.

### Actual behavior
After selecting different languages, the options are double spoken.

### Device information

* Android device: Samsung Galaxy Note 8 (Android 9)
* Fenix version: Firefox Preview Nightly 2/5 @20360606.

**Comment by ekager** [comment]
Is it just reading the Title and then the Subtitle, like "Español, Spanish"? I'm not sure if we should be not reading either of those?

**Comment by sv-sdeiac** [comment]
@ekager 
If the title and the subtitle use Latin characters, the Talkback read both of them. But if the Title is displayed using other characters and the subtitle in Latin characters, only the subtitle is read. 

Anyway, duplicate reading is performed even if both(title/subtitle) or only subtitle is using the Latin characters. Just a little example to be sure that you understand: "Italiano, Italian Italiano, Italian".

Also, I see that those duplicate reading are performed only if specific languages are selected to trigger the Talkback to duplicate. (ex. Tap  Catala -> "Catala, Catalan"; Tap Italiano ->"Italiano, Italian, Italiano, Italian", then all are duplicated.)

**Comment by mcarare** [comment]
This looks similar to https://github.com/mozilla-mobile/fenix/issues/6994.

**Comment by sflorean** [comment]
The issue isn't reproducible anymore on Nightly 29/10.
